common/src/main/res/values/strings.xml

```diff
@@ -1274,4 +1275,7 @@
     <string name="state_update_available">Update available</string>
     <string name="state_up_to_date">Up-to-date</string>
     <string name="sensor_name_health_connect">Health Connect sensors</string>
+    <string name="graph_state_widgets">Statistics widget</string>
```

Comment:
any reason why we have 2 different names for the widget?

Reply:
Just tryed to keep consistent from Home Assistant. The graph look pretty much like it and is called statistics.

Reply:
If you want to change the name to make it more correct thats fine but we should keep it consistent. For example the manage widget screen uses both labels now.

![image](https://github.com/user-attachments/assets/d61b3c15-7b1c-4db0-8182-c0980c1906bf)

Reply:
You are right my friend! I'll then refactor all "graph" -> "stats"